### PR TITLE
Add TargetFormViewController to allow the creation of new targets

### PR DESCRIPTION
**Pull request by smanini1**
Add TargetFormViewController to allow the creation of new targets

#### Description:

Added TargetFormViewController to allow the creation of new targets and SliderAnimationController to handle containerView animations for the form

#### Preview:

ezgif com-video-to-gif (2)

### CODE REVIEW

**Comment by germanStabile** [comment]
There's no doubt SliderAnimationController is very cool. But I think it'd be much more valuable for you to implement this from scratch since you're not hurried with any schedules and this project is purely for learning purposes.

This same effect can be achieved presenting the view controller modally over full context and doing an alpha animation on the background view.

Anyways, those are my two cents the code itself looks good, good job!

File: ios-target/View/Home/TargetFormViewController.swift [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+//
+//  Created by sol manini on 6/26/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+
+class TargetFormViewController: UIViewController {
+  
+  override func viewDidLoad() {
```
Comment (germanStabile) [inline ios-target/View/Home/TargetFormViewController.swift:13]: this is not needed

File: ios-target/View/Home/HomeViewController.swift [hunk 2]
```diff
@@ -28,11 +36,15 @@ class HomeViewController: UIViewController {
...
+    //Sets the presentingViewController transition delegate to the current VC
+    segue.destination.transitioningDelegate = self
+  }
+  
   // MARK: - Actions
   
   @IBAction func tapOnAddTarget(_ sender: Any) {
-    //  TODO:
-    //    targetForm.showTargetForm()
+    //targetForm.showTargetForm()
```
Comment (germanStabile) [inline ios-target/View/Home/HomeViewController.swift:47]: leave the TODO or remove the commented code

Comment (glm4) [inline ios-target/View/Home/HomeViewController.swift:47]: remove commented code.

**Review by glm4 — APPROVED** [review]

**Review by mcousillas6 — APPROVED** [review]

**Pull request merged**